server/heroku/heroku.go

```diff
@@ -74,13 +74,16 @@ func New(e *empire.Empire, authenticator auth.Authenticator) httpx.Handler {
 	// Logs
 	r.Handle("/apps/{app}/log-sessions", &PostLogs{e}).Methods("POST") // hk log
 
-	errorHandler := func(err error, w http.ResponseWriter, r *http.Request) {
-		Error(w, err, http.StatusInternalServerError)
-	}
-
 	api := Authenticate(r, authenticator)
 
-	return middleware.HandleError(api, errorHandler)
+	return httpx.HandlerFunc(func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+		err := api.ServeHTTPContext(ctx, w, r)
+		if err != nil {
+			Error(w, err, http.StatusInternalServerError)
+			reporter.Report(ctx, err)
```

Comment:
Thinking of also making this change. Reporting an error here won't give a very useful stack trace, but it's still good to see that an error was generated, and what the error message was.